Tags all (#7)
PontusHorn/Pico-Tags: README.md

    </article>
{% endfor %}
```

> You can add a side-bar listing documents with the same tags by including something like:

Reviewer: ```suggestion
You can add a sidebar listing documents with the same tags by including something like:
```

```twig
<aside> 
{% for tag in meta.tags %}

>     <h3>Mere under: <a href="{{ base_url }}/tags/?tag={{ tag }}">{{ tag }}</a></h3>

Reviewer: I would remove this line from the example to keep it minimal, even if it's a useful link.

Reviewer: Nevermind, I realised it's working as a heading for each individual tag. Might be better to keep it 👍 
Maybe change the text to be in English instead though? "More pages tagged x" or something like that.

Author: I hadn't even noticed it :) I'll just make the changes online here, instead of pushing it from my clone, all right?

Reviewer: That's perfectly fine by me!

    <ul>
    {% for page in pages if page.title and page.meta.tags %}
        {% if tag in page.meta.tags and not (page.id ends with 'index') and page.id != current_page.id %}
        <li>
            <a href="{{ page.url }}">{{ page.title }}</a> - {{ page.description }}
        </li>
        {% endif %}
    {% endfor %}
    </ul>
{% endfor %}
</aside> 
```

To create a dedicated tags page for either showing a list of available tags or a list of

> documents with a specific tag, use the builtin `tags_all()` function:

Reviewer: ```suggestion
documents with a specific tag, use the builtin `get_all_tags()` function:
```

Create a document `tags.md` in the root of the site:

```
---
Title: Tags
Template: tags
---

Content can go here

```

The tags template can look like this:

```twig
{% extends "index.twig" %}
{% set tag = url_param('tag', 'string') %}

> {% set tags = tags_all() %}

Reviewer: ```suggestion
{% set tags = get_all_tags() %}
```
